cli/sawtooth_cli/sawset.py

```diff
@@ -89,11 +89,12 @@ def _do_config_proposal_create(args):
 
     if args.sabre is not None:
         contract_name, version = parse_sabre_contract_info(args.sabre)
-        txns = [_create_propose_txn_sabre(signer, setting, contract_name, version)
-                                for setting in settings]
+        txns = [_create_propose_txn_sabre(signer, setting,
```

Comment:
These formatting changes should either be broken up into thee own commit or squashed into the previous commit.